src/TeleportJoin.sol

```diff
@@ -253,21 +293,116 @@ contract TeleportJoin {
     /**
     * @dev External function that repays debt with DAI previously pushed to this contract (in general coming from the bridges)
     * @param sourceDomain domain where the DAI is coming from
-    * @param batchedDaiToFlush Amount of DAI that is being processed for repayment
+    * @param targetDomain this domain
```

Comment:
Natspec: `DAI previously pushed to this contract` -> `DAI pulled from the caller`